packages/sources/vue/src/components/selection-controls/VtmnQuantity/VtmnQuantity.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+import '@vtmn/css-quantity/dist/index-with-vars.css';
+import { computed, defineComponent, PropType, reactive } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnQuantity',
+  components: { VtmnButton },
+  props: {
+    modelValue: {
+      type: Number as PropType<number>,
+      default: 0,
+    },
+    id: {
```

Comment:
In order to avoid confusion with the HTML id attribute when we call this component in the parent I suggest to call this props 'identifier'.

What do you think about it ?

Reply:
seems right, we did this for other components